src/components/Settings/Settings.js

```diff
@@ -0,0 +1,198 @@
+  "preferred_photo_license", "preferred_sound_license", "make_observation_licenses_same",
+  "make_photo_licenses_same", "make_sound_licenses_same",
+  ...Object.values( EMAIL_NOTIFICATIONS )
+];
+
+type Props = {
+  children: React.Node,
+}
+
+
+const SettingsTabs = ( { activeTab, onTabPress } ): React.Node => {
+  return (
+    <>
+      <View style={[viewStyles.tabsRow, viewStyles.shadow]}>
+        <Pressable onPress={() => onTabPress( TAB_TYPE_PROFILE )} accessibilityRole="link">
+          <Text style={activeTab === TAB_TYPE_PROFILE ? textStyles.activeTab : null}>profile</Text>
+        </Pressable>
+        <Pressable onPress={() => onTabPress( TAB_TYPE_ACCOUNT )} accessibilityRole="link">
+          <Text style={activeTab === TAB_TYPE_ACCOUNT ? textStyles.activeTab : null}>account</Text>
+        </Pressable>
+        <Pressable onPress={() => onTabPress( TAB_TYPE_NOTIFICATIONS )} accessibilityRole="link">
+          <Text style={activeTab === TAB_TYPE_NOTIFICATIONS ? textStyles.activeTab : null}>notifications</Text>
+        </Pressable>
+        <Pressable onPress={() => onTabPress( TAB_TYPE_RELATIONSHIPS )} accessibilityRole="link">
+          <Text style={activeTab === TAB_TYPE_RELATIONSHIPS ? textStyles.activeTab : null}>relationships</Text>
+        </Pressable>
+        <Pressable onPress={() => onTabPress( TAB_TYPE_CONTENT_DISPLAY )} accessibilityRole="link">
+          <Text style={activeTab === TAB_TYPE_CONTENT_DISPLAY ? textStyles.activeTab : null}>content&display</Text>
+        </Pressable>
+        <Pressable onPress={() => onTabPress( TAB_TYPE_APPLICATIONS )} accessibilityRole="link">
+          <Text style={activeTab === TAB_TYPE_APPLICATIONS ? textStyles.activeTab : null}>applications</Text>
+        </Pressable>
+      </View>
+    </>
+  );
+};
+
+const Settings = ( { children }: Props ): Node => {
+  const [activeTab, setActiveTab] = useState( TAB_TYPE_PROFILE );
+  const [settings, setSettings] = useState( {} );
+  const [accessToken, setAccessToken] = useState( );
+  const [isLoading, setIsLoading] = useState( true );
+  const [isSaving, setIsSaving] = useState( false );
+  const [authorizedApps, setAuthorizedApps] = useState( [] );
+  const [providerAuthorizations, setProviderAuthorizations] = useState( [] );
+
+  useEffect( () => {
+    let isCurrent = true;
+
+    if ( isCurrent ) {
+      getAPIToken( true ).then( setAccessToken );
+    }
+
+    return ( ) => {
+      isCurrent = false;
+    };
+  }, [] );
+
+
+  const fetchProfile = useCallback( async () => {
+    try {
+      const response = await inatjs.users.me( {api_token: accessToken} );
+      console.log( "User object", response.results[0] );
+      setSettings( response.results[0] );
+      setIsLoading( false );
+    } catch ( exc ) {
+      console.error( "exc", exc );
+      console.error( JSON.stringify( exc ) );
+    }
+  }, [accessToken] );
+
+
+  useEffect( () => {
```

Comment:
I'm getting a 404 error with this API call when testing with staging API v2. I also think this should be moved into `SettingsApplications.js`, because we don't need to make this API call or load authorized applications at all unless a user taps that tab.

The `authorizedApps` and `providerAuthorizations` state and `revokeApp` functions can also be moved into the `SettingsApplications` component.

Reply:
Fixed in latest commit. However, in API v2 there are not endpoints to pull authorized apps and provider authorizations.